### PR TITLE
 K8SPXC-257 tests

### PR DESCRIPTION
**Pull request by blez**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | failed |
| big-data | passed |
| init-deploy | passed |
| scaling | passed |
| recreate | passed |
| scaling-proxysql | passed |
| limits | passed |
| self-healing | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | passed |
| monitoring | passed |
| upgrade-consistency | passed |
| security-context | passed |
| self-healing-advanced | failed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| affinity | passed |
| operator-self-healing | passed |
| one-pod | passed |
| auto-tuning | passed |
| scheduled-backup | failed |
| proxysql-sidecar-res-limits | failed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| storage | failed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| big-data | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| self-healing | passed |
| self-healing-advanced | failed |
| operator-self-healing | passed |
| demand-backup-encrypted-with-tls | passed |

File: deploy/cr.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ spec:
   sslInternalSecretName: my-cluster-ssl-internal
   allowUnsafeConfigurations: false
 #  pause: false
-  updateStrategy: SmartUpdate
+#  updateStrategy: RollingUpdate
```
Comment (delgod) [inline deploy/cr.yaml:16]: please revert default

File: e2e-tests/monitoring/compare/statefulset_monitoring-pxc-oc.yml [hunk 2]
```diff
@@ -163,6 +163,17 @@ spec:
         - mountPath: /etc/mysql/vault-keyring-secret
           name: vault-keyring-secret
       dnsPolicy: ClusterFirst
```
Comment (delgod) [inline e2e-tests/monitoring/compare/statefulset_monitoring-pxc-oc.yml:165]: I think entrypoint.sh also needed for this file

File: e2e-tests/monitoring-2-0/compare/statefulset_monitoring-pxc-oc.yml [hunk 3]
```diff
@@ -170,6 +170,17 @@ spec:
         - mountPath: /etc/mysql/vault-keyring-secret
           name: vault-keyring-secret
       dnsPolicy: ClusterFirst
```
Comment (delgod) [inline e2e-tests/monitoring-2-0/compare/statefulset_monitoring-pxc-oc.yml:172]: I think entrypoint.sh also needed for this file

File: deploy/cr.yaml (hunk 1 above)
Comment (blez) [reply deploy/cr.yaml:16]: But this is default

Comment (blez) [reply deploy/cr.yaml]: https://github.com/percona/percona-xtradb-cluster-operator/pull/408/commits/29335ef3ef25187e4bdfcebcccd1774c6748678c

File: e2e-tests/monitoring/compare/statefulset_monitoring-pxc-oc.yml (hunk 2 above)
Comment (blez) [reply e2e-tests/monitoring/compare/statefulset_monitoring-pxc-oc.yml:169]: https://github.com/percona/percona-xtradb-cluster-operator/pull/408/commits/84533f38db5e68f75064af81973a65cea1cde512

File: e2e-tests/monitoring-2-0/compare/statefulset_monitoring-pxc-oc.yml (hunk 3 above)
Comment (blez) [reply e2e-tests/monitoring-2-0/compare/statefulset_monitoring-pxc-oc.yml:176]: https://github.com/percona/percona-xtradb-cluster-operator/pull/408/commits/84533f38db5e68f75064af81973a65cea1cde512

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| big-data | passed |
| storage | failed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-encrypted-with-tls | passed |
| self-healing | passed |
| self-healing-advanced | failed |
| operator-self-healing | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| big-data | failed |
| init-deploy | failed |
| storage | failed |
| recreate | passed |
| scaling-proxysql | passed |
| limits | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| monitoring | failed |
| upgrade | passed |
| monitoring-2-0 | passed |
| self-healing-advanced | failed |
| upgrade-consistency | passed |
| affinity | passed |
| demand-backup | passed |
| security-context | passed |
| operator-self-healing | passed |
| one-pod | passed |
| auto-tuning | passed |
| scheduled-backup | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| recreate | passed |
| storage | failed |
| scaling | passed |
| init-deploy | failed |
| limits | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| monitoring | passed |
| upgrade | passed |
| self-healing-advanced | passed |
| monitoring-2-0 | failed |
| upgrade-consistency | passed |
| security-context | passed |
| affinity | passed |
| demand-backup | passed |
| operator-self-healing | passed |
| one-pod | passed |
| auto-tuning | passed |
| scheduled-backup | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | failed |
| scaling | passed |
| recreate | passed |
| init-deploy | passed |
| big-data | failed |
| scaling-proxysql | passed |
| limits | passed |
| self-healing | failed |
| upgrade | passed |
| restore-to-encrypted-cluster | passed |
| monitoring | passed |
| upgrade-consistency | passed |
| self-healing-advanced | passed |
| security-context | passed |
| monitoring-2-0 | failed |
| demand-backup | passed |
| affinity | passed |
| operator-self-healing | passed |
| one-pod | passed |
| auto-tuning | passed |
| scheduled-backup | failed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | failed |

**Pull request merged**